### PR TITLE
changelog for 5.13.0

### PR DESCRIPTION
**Pull request by jburel**
cc @sbesson @will-moore

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 5.13.0 (November 2022)
+
+## New features
+
+- Remove Anonymous Diffie-Hellman default configuration [#336](https://github.com/ome/omero-py/pull/336)
+- Publish API doc [#330](https://github.com/ome/omero-py/pull/330)
```
Comment (sbesson) [inline CHANGELOG.md:6]: Minor suggestion: https://github.com/ome/omero-py/blob/master/setup.py#L205 could be updated to also include the new API documentation location?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# 5.13.0 (November 2022)
+
+## New features
+
+- Remove Anonymous Diffie-Hellman default configuration [#336](https://github.com/ome/omero-py/pull/336)
```
Comment (sbesson) [inline CHANGELOG.md:5]: Is there a place that explains that the certificates MUST be configured server-side when upgrading toh this version of the OMERO.py client? Should this belong to the release notes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+- Publish API doc [#330](https://github.com/ome/omero-py/pull/330)
+- Switch to new output command (GHA) [#340](https://github.com/ome/omero-py/pull/340)
```
Comment (sbesson) [inline CHANGELOG.md:7]: This does not feel like a feature but it's not really a bug fix either. We have used `Other updates` in the past, should we use this for general maintenance category?

**Review by sbesson — COMMENTED** [review]
A few comments. Probably the biggest question is around being are explicit about the server steps required to work with the changes in #336

File: CHANGELOG.md (hunk 3 above)
Comment (jburel) [reply CHANGELOG.md:7]: I will put it under ``other updates``

File: CHANGELOG.md (hunk 2 above)
Comment (jburel) [reply CHANGELOG.md]: I have added info into changelog. But I think I will also add line in README

File: CHANGELOG.md (hunk 1 above)
Comment (jburel) [reply CHANGELOG.md:6]: done

**Review by sbesson — APPROVED** [review]

**Pull request merged**